Add variable to extend the CPPFLAGS. When specifying CPPFLAGS, it will 
replace CPPFLAGS. When giving TARGET_CPPFLAGS it is added to the 
CPPFLAGS.

--- a/Make.Rules
+++ b/Make.Rules
@@ -83,7 +83,7 @@ WARNINGS=-Wall -Wwrite-strings -Wpointer
 COPTS ?= -O2
 CFLAGS ?= $(COPTS) $(DEBUG)
 CFLAGS += $(WARNINGS)
-CPPFLAGS += -Dlinux $(DEFINES) $(LIBCAP_INCLUDES)
+CPPFLAGS += $(TARGET_CPPFLAGS) -Dlinux $(DEFINES) $(LIBCAP_INCLUDES)
 LDFLAGS ?= # -g
 
 BUILD_CC ?= $(CC)
